service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java

```diff
@@ -29,6 +33,32 @@ public static ApiCreateWorkspaceRequestBody createWorkspaceRequestBody() {
     return createWorkspaceRequestBody(ApiWorkspaceStageModel.MC_WORKSPACE);
   }
 
+  public static Workspace createWorkspace(
+      @Nullable UUID workspaceUuid, WorkspaceStage workspaceStage) {
+    return defaultWorkspaceBuilder(workspaceUuid).workspaceStage(workspaceStage).build();
+  }
+
```

Comment:
nit: it seems like `createMcWorkspace(UUID)` is also a common pattern and might be useful

Reply:
sg